lib/MusicBrainz/Server/Controller/Role/Collection.pm

```diff
@@ -33,28 +33,38 @@ role
 
     method _all_collections => sub {
         my ($self, $c) = @_;
-        return [ $c->model('Collection')->find_all_by_entity($entity_type, $c->stash->{$entity_name}->id) ];
+        my ($collections) = $c->model('Collection')->find_by({
+            entity_id => $c->stash->{$entity_name}->id,
+            entity_type => $entity_type,
+        });
```

Comment:
It changed slightly 'cause it used to use `query_to_list` and now uses `query_to_list_limited`. `query_to_list` just returned a list of all results directly, but `query_to_list_limited` returns `($array_ref_of_results, $total_results)`, which is why it's doing `my ($collections) = ` instead of `my @collections) = ` (we don't care about the total in the first case).